host-controller/src/main/java/org/jboss/as/domain/controller/operations/HostProcessReloadHandler.java

```diff
@@ -76,10 +76,13 @@
             .setAlternatives(ModelDescriptionConstants.USE_CURRENT_DOMAIN_CONFIG)
             .build();
 
+    protected static final AttributeDefinition TIMEOUT = new SimpleAttributeDefinitionBuilder(ModelDescriptionConstants.TIMEOUT, ModelType.INT, true)
+            .setDefaultValue(new ModelNode(0)).build();
```

Comment:
I think for this case having a 0 by default for the timeout could not be great idea because after this PR, the user is not able to configure different suspend values for a reload operation. That means if we use 0 as default, doing something like this (suppose we are in a domain mode with on HC and two managed servers):
`/host=master/server-config=server-one:suspend(timeout=60)`
`/host=master/server-config=server-one:suspend(timeout=120)`
`/host=master:reload`
Last reload will automatically overwrite the previous suspend configured timeout with 0. 

The user can use now:
`/host=master:reload(timeout=120)`
But he is not longer able to do the reload with different suspend timeouts configured for each managed server